recipe/builder.py

```diff
@@ -14,13 +14,25 @@ def install():
     """ populate the environment variable added by the patch and actually
         install with pip
     """
+
+    cuda_version = os.environ.get('cuda_compiler_version', None)
+
+    try:
+        float(cuda_version)
+        is_cuda = True
+    except (TypeError, ValueError):
+        is_cuda = False
+
     if WIN:
         cmake_args = (
             "-DDLIB_FORCE_MSVC_STATIC_RUNTIME=OFF\n"
             "-DDLIB_LINK_WITH_SQLITE3=OFF\n"
             "-DBUILD_SHARED_LIBS=ON\n"
             "-DDLIB_IN_PROJECT_BUILD=ON\n"
-        ).format(os.environ)
+            "-DCUDNN_INCLUDE_DIR:PATH={LIBRARY_INC}\n"
```

Comment:
[CI says `LIBRARY_INC` is undefined]( https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=855184&view=logs&jobId=171a126d-c574-5c8c-1269-ff3b989e923d&j=171a126d-c574-5c8c-1269-ff3b989e923d&t=623c2ba8-c4f1-582d-6c78-1c7699e2e0a2 ), which is odd as [it should be available]( https://docs.conda.io/projects/conda-build/en/latest/user-guide/environment-variables.html ). Might be worth dumping `os.environ` to learn what is set

Reply:
Nvm think you already fixed it 😅